### PR TITLE
DOC: readme tweaks

### PR DESCRIPTION
**Pull request by bmorris3**
The README suggests displaying the app with `specviz.app`, when we usually recommend `specviz.show()`, so I updated it.

Also, @orifox once requested that we center-align the logo. I tried to sort that out, but it looks like github still doesn't fully support the necessary rst features.

### Change log entry

- [X] Is a change log needed? If yes, is it added to `CHANGES.rst`? If you want to avoid merge conflicts,
  list the proposed change log here for review and add to `CHANGES.rst` before merge. If no, maintainer
  should add a `no-changelog-entry-needed` label.

### Checklist for package maintainer(s)

This checklist is meant to remind the package maintainer(s) who will review this pull request of some common things to look for. This list is not exhaustive.

- [X] Are two approvals required? Branch protection rule does not check for the second approval. If a second approval is not necessary, please apply the `trivial` label.
- [ ] Do the proposed changes actually accomplish desired goals? Also manually run the affected example notebooks, if necessary.
- [X] Do the proposed changes follow the [STScI Style Guides](https://github.com/spacetelescope/style-guides)?
- [X] Are tests added/updated as required? If so, do they follow the [STScI Style Guides](https://github.com/spacetelescope/style-guides)?
- [X] Are docs added/updated as required? If so, do they follow the [STScI Style Guides](https://github.com/spacetelescope/style-guides)?
- [ ] Did the CI pass? If not, are the failures related?
- [ ] Is a milestone set? Set this to bugfix milestone if this is a bug fix and needs to be released ASAP; otherwise, set this to the next major release milestone.
- [X] After merge, any internal documentations need updating (e.g., JIRA, Innerspace)?

### CODE REVIEW

**Comment by pllim** [comment]
I don't think PyPA likes your README. 😬

**Comment by pllim** [comment]
As for the content, seems weird to mention Specviz but then point to Cubeviz notebook.

And should add mention of Specviz2D while you are at it?

**Comment by bmorris3** [comment]
Just finding out the `raw` directive is forbidden on PyPI... investigating.

File: README.rst [hunk 1]
```diff
@@ -22,6 +17,11 @@
     :target: https://www.astropy.org
     :alt: Powered by Astropy
 
+.. image:: docs/logos/jdaviz.svg
```
Comment (pllim) [inline README.rst:20]: I think this should be before the badges, as before?

**Comment by bmorris3** [comment]
Experimenting some more. The centering issue is blocked by https://github.com/github/markup/issues/163 – I tried a few work arounds, none worked.

**Comment by bmorris3** [comment]
I have resolved to ditch the centering effort. This PR now has my tiniest jdaviz diff ever!

**Review by kecnry — APPROVED** [review]
suuuuper controversial now, but I think we can live with it 😉

**Comment by pllim** [comment]
While you are at it, might as well point the notebook mention to be consistent with the import mention, and add Specviz2D?

**Comment by bmorris3** [comment]
> add Specviz2D?

I'm not sure I follow the 2D part. If we change the links from Cubeviz -> Specviz, then we're only mentioning Specviz. Should we additionally mention Specviz2D, or all of the other helpers too?

**Comment by pllim** [comment]
I took the liberty to add Specviz2D in a second commit. If you don't like it, feel free to drop it. Thanks!

File: README.rst [hunk 2]
```diff
@@ -96,10 +96,14 @@ The power of ``jdaviz`` is that it can integrated into your Jupyter notebook wor
...
-    specviz.app
+    specviz.show()
 
 To learn more about the various ``jdaviz`` application configurations and loading data, see the
-`specviz <https://jdaviz.readthedocs.io/en/latest/specviz/import_data.html>`_, `cubeviz <https://jdaviz.readthedocs.io/en/latest/cubeviz/import_data.html>`_, `mosviz <https://jdaviz.readthedocs.io/en/latest/mosviz/import_data.html>`_, or `imviz <https://jdaviz.readthedocs.io/en/latest/imviz/import_data.html>`_ tools.
+`Specviz <https://jdaviz.readthedocs.io/en/latest/specviz/import_data.html>`_,
+`Cubeviz <https://jdaviz.readthedocs.io/en/latest/cubeviz/import_data.html>`_,
+`Mosviz <https://jdaviz.readthedocs.io/en/latest/mosviz/import_data.html>`_,
+`Imviz <https://jdaviz.readthedocs.io/en/latest/imviz/import_data.html>`_,
+or `Specviz2d <https://jdaviz.readthedocs.io/en/latest/specviz2d/import_data.html>`_ tools.
```
Comment (pllim) [inline README.rst:106]: @duytnguyendtn said the D is also capitalized.

```suggestion
or `Specviz2D <https://jdaviz.readthedocs.io/en/latest/specviz2d/import_data.html>`_ tools.
```

**Review by pllim — APPROVED** [review]
Thanks!

**Pull request merged**